src/mongocxx/pool.cpp

```diff
@@ -39,20 +39,20 @@ void pool::_release(client* client) {
 
 pool::~pool() = default;
 
-pool::pool(const uri& uri, stdx::optional<options::ssl> ssl_options)
+pool::pool(const uri& uri, const options::pool& options)
```

Comment:
Nope, `options` has a default argument of `options::pool()`.  See changes to `mongocxx/pool.hpp` in the same commit:

```
+    explicit pool(const uri& mongodb_uri = mongocxx::uri(),
+                  const options::pool& options = options::pool());
```

This makes the signature of the pool constructor more in line with the [signature of the client constructor](https://github.com/mongodb/mongo-cxx-driver/blob/r3.0.2/src/mongocxx/client.hpp#L69).